docs/guide/terraform-guide.md

```diff
@@ -0,0 +1,181 @@
+title: Sematext Terraform Guide
+description: Sematext Cloud supplies a Terraform Provider and Resources used to devop monitoring across all your systems, Apps, and services.
+
+You can now include Sematext Cloud as your go-to choice for fast deploy of visualization, APM, metrics and logging aspects within your cloud-as-code solutions using proven managed sub-components.
+
+Sematext is now a Terraform Verified Partner! You can access the Sematext Cloud Provider in the
```

Comment:
I think we can drop announcement-like wording, because it will stop being news "tomorrow".

Reply:
Removed.